examples/utils/example-utils/src/migrationTool/sameContainerMigrationTool.ts

```diff
@@ -447,7 +371,7 @@ export class SameContainerMigrationTool extends DataObject implements ISameConta
 				// random ack.  Probably means storing the Quorum code accept sequence number and verifying the summary is based on that sequence number.
 				// Would be good if we can verify the contents somehow too.
 				// TODO: Not appropriate to be watching _seenV1SummaryAck here, I'm just doing this to simulate second ack after acceptance
-				if (this._v1SummaryDone && op.type === MessageType.SummaryAck) {
+				if (this.acceptedSeqNum !== undefined && op.type === MessageType.SummaryAck) {
```

Comment:
@ChumpChief, does checking if `acceptedSeqNum` is defined make sense here? I'm thinking we can probably just remove it all together since the v1 summary is gone, but just to be safe I thought it would be good to make sure we have at least accepted the proposal

Reply:
Agree this conditional can drop the seq num check - maybe add an assert just inside to verify that it's defined (vs. some weird spurious ack that we don't expect)